src/radarChart.ts

```diff
@@ -340,13 +346,13 @@ module powerbi.extensibility.visual {
                     }
 
                     if (source.objects) {
-                        color = colorHelper.getColorForMeasure(source.objects, queryName);
+                        color = localColorHelper.getColorForMeasure(source.objects, queryName);
                     }
                 }
 
-                legendData.dataPoints.push({
+                legendData.dataPoints.push(<LegendDataPoint>{
                     label: displayName,
-                    color: color,
+                    color: colorHelper.isHighContrast ? colorHelper.getHighContrastColor("foreground", color) : color,
```

Comment:
Create a new variable to make code cleaner instead of adding too long code lines